osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -288,17 +288,15 @@ protected override void OnHoverLost(HoverLostEvent e)
         private class ScoreComponentLabel : Container, IHasTooltip
         {
             private const float icon_size = 20;
-
-            private readonly string name;
             private readonly FillFlowContainer content;
 
             public override bool Contains(Vector2 screenSpacePos) => content.Contains(screenSpacePos);
 
-            public string TooltipText => name;
+            public string TooltipText { get; }
 
             public ScoreComponentLabel(LeaderboardScoreStatistic statistic)
             {
-                name = statistic.Name;
+                TooltipText = statistic.Name;
```

Comment:
These classes are very small; the field isn't used elsewhere, and doesn't has a special meaning like `isDisposed`. So I think it's fine to lose the name of the field.

Reply:
I sort of disagree with these sorts of renames, would rather rename properties back to `Name` and add delegating explicit `IHasTooltip` implementations.